Doc/c-api/structures.rst

```diff
@@ -187,23 +187,23 @@ Implementing functions and methods
                                         PyObject *kwargs);
 
 
-.. c:type:: _PyCFunctionFast
+.. c:type:: PyCFunctionFast
 
    Type of the functions used to implement Python callables in C
    with signature :c:macro:`METH_FASTCALL`.
    The function signature is::
 
-      PyObject *_PyCFunctionFast(PyObject *self,
+      PyObject *PyCFunctionFast(PyObject *self,
```

Comment:
Nit: I think it's better to keep the below lines aligned like before. And there are several similar issues in this change.